openupgrade_scripts/scripts/account/14.0.1.1/post-migration.py

```diff
@@ -546,6 +546,17 @@ def fill_statement_lines_with_no_move(env):
         st_line.move_id.with_context(skip_account_move_synchronization=True).write(
             to_write
         )
+    if st_lines:
+        openupgrade.logged_query(
+            env.cr,
+            """
+            UPDATE account_move am
+            SET partner_bank_id = absl.bank_account_id
+            FROM account_bank_statement_line absl
+            WHERE am.statement_line_id IN %s AND absl.bank_account_id IS NOT NULL
+                AND am.partner_bank_id IS NULL""",
+            (tuple(st_lines.ids),),
```

Comment:
Why not doing the sentence in pre here and remove the one in pre?

Reply:
ok